src/MoBi.Assets/AppConstants.cs

```diff
@@ -283,17 +283,17 @@ public static string DisplayValue(double displayValue, string displayUnit)
 
          public static string CommitingChangesToModulesMessage(ModuleConfiguration moduleConfiguration, bool hasMoleculeChanges, bool hasParameterChanges)
          {
-            string message = $"Changes will be applied in the module <i>{moduleConfiguration.Module.Name}</i>:{Environment.NewLine}";
+            string message = $"Changed parameter Values will be committed to the PV of the module <i>{moduleConfiguration.Module.Name}</i>:{Environment.NewLine}";
```

Comment:
Pavel wrote the comments with contractions and with mixed case but this should be fixed in the caption

Don't use PV or IC contractions and usually when we mean building block we have to spell it out as "Parameter Values Building Block".

this also says "parameter Values" where Values has the wrong case.

Make sure this is consistent for all three of the lines changed here.